shishir/Day1/activity_main.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#3b5998"
+    tools:context="com.example.myapplicationapplication.MainActivity"
+    tools:layout_editor_absoluteY="81dp"
```

Comment:
add padding to your parent element (in this case constraint layout) to ensure padding to all elements within:
`android:padding="20dp"`